sdk/trace/src/main/java/io/opentelemetry/sdk/trace/AndroidFriendlyRandomIdGenerator.java

```diff
@@ -32,11 +32,10 @@ public String generateSpanId() {
   @Override
   public String generateTraceId() {
     long idHi;
-    long idLo;
     do {
       idHi = random.nextLong();
-      idLo = random.nextLong();
-    } while (idHi == INVALID_ID && idLo == INVALID_ID);
+    } while (idHi == INVALID_ID);
```

Comment:
Is this correct? I thought for 64-bit-traceid interop, that we required the low bytes to be non-zero in all cases, rather than the high bytes.

Reply:
This makes sense to me - can't find anything in the spec but will check on that.